project/Build.scala

```diff
@@ -79,7 +83,9 @@ object DottyBuild extends Build {
 
       tuning ::: agentOptions ::: travis_build ::: fullpath
     }
-  )
+  ) ++ (if (isTravisBuild) Seq() else addCommandAlias("test", "partest"))
```

Comment:
The `test` key only generates partest files, it doesn't run the tests. So the alias prevents people from having to remember to use `partest` instead of test.